### ISSUE TITLE
5.7.8 Username and Password not accepted

### ISSUE THREAD

**Issue by ahdee** [magepal/magento2-gmailsmtpapp#43]
Hi I tried testing email but keep getting this message. I've done the following. 
1) no 2 step authentication
2) allow  less secure apps
3) turn off captcha

tried both TLS and SSL but still the same error.  Is there a way around this? 
thanks! 
A

**Comment by srenon** [comment]
@ahdee .... Could you provide the exact error msg?

**Comment by ahdee** [comment]
Hi Srenon: yes here it is: 
"Result 5.7.8 Username and Password not accepted. Learn more at 5.7.8 https://support.google.com/mail/?p=BadCredentials l3sm174288777pgc.41 - gsmtp"

**Comment by srenon** [comment]
The error msg does not have anything to do with the port (ssl or tls). gmail is rejecting because the password it is getting is incorrect.
 
Are you 100% sure your password is correct? Can you manually login?

Before saving your email password (in system config) try doing a test email

**Comment by ahdee** [comment]
Srenon: it works now, thanks. Ok here is the error.  Its not that I entered the wrong password.  Strange thing is that "testing email" fails every time I save and flush the cache.  If I entered the password, turn off captcha and send it works.  Then I saved the config, however even if the test email does not work after that, my contact us page works; I'm not sure what the issue is, but I glad it works now. thank you.
